configure

```diff
@@ -577,8 +577,42 @@ int main() {
   except:
     error ('unexpected exception!' + configure_log_hint)
   else:
-    break
+    max_align_t_defined = False
+    report ('Checking for ::max_align_t: ')
+    try:
+      compile ('''
+#include <cstddef>
```

Comment:
Whoops... Must have forgotten to put it back in after testing on my system (since I can't get the tests to fail any other way here...).
Fix committed.